solana/ts/sdk/ntt.ts

```diff
@@ -0,0 +1,799 @@
+import { type ChainName, toChainId, coalesceChainId, type ChainId, SignedVaa } from '@certusone/wormhole-sdk'
+import { serializeLayout, toChainId as SDKv2toChainId } from "@wormhole-foundation/sdk-base";
+import { deserialize } from "@wormhole-foundation/sdk-definitions";
+//TODO these need to be properly exported by the SDK
+import { nttManagerMessageLayout, NttManagerMessage, nativeTokenTransferLayout } from "@wormhole-foundation/sdk-definitions/dist/cjs/payloads/ntt";
```

Comment:
we're not exporting layouts for payloads directly, should we?

We've been exporting VAA specific stuff from the Protocols namespace 

https://github.com/wormhole-foundation/connect-sdk/blob/main/core/definitions/src/protocols/tokenBridge.ts#L29-L61

Reply:
these are now exported